routes/auth.js

```diff
@@ -0,0 +1,52 @@
+const router = require('express').Router();
+const User = require('../models/User');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+
+router.post('/register', async (req, res) => {
+    const saltRounds = 10;
```

Comment:
why saltRounds=10?

Reply:
It was the value recommended online. From what I understand: 
the lower it is, the more vulnerable the password becomes
The higher it is, the more time it takes to generate the hashed password

Reply:
Perfect, I trust your judgment, we can always change it au pire